src/SFA.DAS.AdminService.Web/Controllers/Roatp/Apply/RoatpGatewayPeopleInControlController.cs

```diff
@@ -48,6 +48,28 @@ public async Task<IActionResult> EvaluatePeopleInControlPage(PeopleInControlPage
 
                 return await SubmitGatewayPageAnswer(viewModel, $"{GatewayViewsLocation}/PeopleInControl.cshtml");
             }
+
+
+            [HttpGet("/Roatp/Gateway/{applicationId}/Page/PeopleInControlRisk")]
+            public async Task<IActionResult> GetGatewayPeopleInControlRiskPage(Guid applicationId, string pageId)
+            {
+                var username = _contextAccessor.HttpContext.User.UserDisplayName();
+                var viewModel = await _orchestrator.GetPeopleInControlHighRiskViewModel(new GetPeopleInControlHighRiskRequest(applicationId, username));
+                return View($"{GatewayViewsLocation}/PeopleInControlHighRisk.cshtml", viewModel);
+            }
+
+        [HttpPost("/Roatp/Gateway/{applicationId}/Page/PeopleInControlRisk")]
+        public async Task<IActionResult> EvaluatePeopleInControlHighRiskPage(PeopleInControlHighRiskPageViewModel viewModel)
+        {
+            var username = _contextAccessor.HttpContext.User.UserDisplayName();
+            var vmRebuild = await _orchestrator.GetPeopleInControlHighRiskViewModel(new GetPeopleInControlHighRiskRequest(viewModel.ApplicationId, username));
+            viewModel.CompanyDirectorsData = vmRebuild?.CompanyDirectorsData;
+            viewModel.PscData = vmRebuild?.PscData;
+            viewModel.TrusteeData = vmRebuild?.TrusteeData;
+            viewModel.WhosInControlData = vmRebuild?.WhosInControlData;
```

Comment:
Is this only required when there are validation errors?  I wonder if it's be better creating an overload of SubmitGatewayPageAnswer so the call is only made if there are errors rather than all the time.

Reply:
This is a good idea,  Want to discuss in DevCuddle maybe, and figure out how to implement (this story? tech task?)

Reply:
I'll push this branch as is, and investigate this in its own branch or something like that

Reply:
yeah I addressed this anyway